sdk/core/core-rest-pipeline/src/policies/exponentialRetryPolicy.ts

```diff
@@ -48,7 +48,7 @@ export function exponentialRetryPolicy(
       }),
     ],
     {
-      maxRetries: options.maxRetries ?? 10,
+      maxRetries: options.maxRetries ?? 3,
```

Comment:
can we use the constant `DEFAULT_RETRY_POLICY_COUNT` instead?

Reply:
yes! thank you 🙏